borg/archive.py

```diff
@@ -180,7 +180,7 @@ def _load_meta(self, id):
     def load(self, id):
         self.id = id
         self.metadata = self._load_meta(self.id)
-        decode_dict(self.metadata, (b'name', b'hostname', b'username', b'time', b'time_end'))
+        decode_dict(self.metadata, (b'name', b'comment', b'hostname', b'username', b'time', b'time_end'))
```

Comment:
good point. can you create a ticket about this. or maybe even a general one like "put all constants into one place and deduplicate them"?